src/managed/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs

```diff
@@ -214,6 +231,11 @@ private static Platform DetermineOSPlatform()
             {
                 return Platform.Darwin;
             }
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD")))
```

Comment:
Why is this different from the other platforms? I mean, why not OSPlatform.FREEBSD instead of OSPlatform.Create("FREEBSD")?

Reply:
That was rejected by  API review board.  https://github.com/dotnet/corefx/issues/23989 and https://github.com/dotnet/corefx/pull/23986
For now, they did not want to add FreeBSD member.

Reply:
Ah, ok, that makes sense.